scripts-dev/convert_server_keys.py

```diff
@@ -54,15 +53,9 @@ def convert_v1_to_v2(server_name, valid_until, keys, certificate):
         "server_name": server_name,
         "verify_keys": {key_id: {"key": key} for key_id, key in keys.items()},
         "valid_until_ts": valid_until,
-        "tls_fingerprints": [fingerprint(certificate)],
```

Comment:
I don't think this is necessary, but on the other hand, I don't think this script is useful any more.